msgcntr/messageforums-component-impl/src/java/org/sakaiproject/component/app/messageforums/PermissionLevelManagerImpl.java

```diff
@@ -534,6 +534,8 @@ else if (customPermName.equals(PermissionLevel.MARK_AS_READ))
     		return permissionLevel.getMarkAsRead();
     	else if (customPermName.equals(PermissionLevel.MODERATE_POSTINGS))
     		return permissionLevel.getModeratePostings();
+    	else if (customPermName.equals(PermissionLevel.IDENTIFY_ANON_AUTHORS))
```

Comment:
I realize the surrounding code doesn't follow the best practise of using braces for single line conditionals, but we shouldn't introduce more occurrences of a bad practise.